tower/src/builder/mod.rs

```diff
@@ -459,6 +459,22 @@ impl<L> ServiceBuilder<L> {
     {
         self.layer.layer(service)
     }
+
+    /// Wrap the async function `F` with the middleware provided by this [`ServiceBuilder`]'s
+    /// [`Layer`]'s, returning a new [`Service`].
+    ///
+    /// This is a convenience for doing `.service(service_fn(handler_function))`.
```

Comment:
nit: might be nice if this linked to `service_fn` someplace?

Reply:
Good idea. I made `.service(service_fn(handler_function))` link to it. Doesn't think that totally makes sense but not sure where else to put it 🤷